pygfx/show/loop.py

```diff
@@ -0,0 +1,47 @@
+"""
+Short utilities to quickly visualize scenes without any boilerplate.
+"""
+from ..renderers import WgpuRenderer
+from ..cameras import PerspectiveCamera
+from ..controls import OrbitControls
+
+from .canvases import get_interactive_canvas_cls
+
+
+def run(backend, canvas_cls, extras, scene, animate=None, interactive=True):
+    """
+    Render a given scene and optional animate callback in the given backend GUI layer
+    """
+    if backend == "qt":
+        QtWidgets, _ = extras  # noqa: N806
+        app = QtWidgets.QApplication([])
+
+    camera = PerspectiveCamera(70, 16 / 9)
+    camera.position.z = 500

```

Comment:
These numbers are arbitrary ...

Reply:
I think the camera should also be an argument to `show()`.

Reply:
or ... the renderer/scene could detect the bounding box of the total scene and place the camera such that the whole scene is visible.